File: ios/RCTMGL-v10/RCTMGLCameraManager.m
#import "React/RCTBridgeModule.h"
#import <React/RCTViewManager.h>
#import <Foundation/Foundation.h>

@interface RCT_EXTERN_MODULE(RCTMGLCameraManager, RCTViewManager)

RCT_EXPORT_VIEW_PROPERTY(stop, NSDictionary)
RCT_EXPORT_VIEW_PROPERTY(defaultStop, NSDictionary)
RCT_EXPORT_VIEW_PROPERTY(followUserLocation, BOOL)

@end

File: ios/RCTMGL-v10/RCTMGLCameraManager.swift
import Foundation
import MapboxMaps

@objc(RCTMGLCameraManager)
class RCTMGLCameraManager  : RCTViewManager {
    @objc
    override static func requiresMainQueueSetup() -> Bool {
        return true
    }
    
    override func view() -> UIView! {
        return RCTMGLCamera()
    }
}
